### ISSUE TITLE
Offline Transaction Signer not found in latest version

### ISSUE THREAD

**Issue by l3utterfly** [Nethereum/Nethereum#716]
Even though the documentation was updated to use the static "OfflineTransactionSigner" method, it is still not found.

I have used the static version as specified in the documentation: https://docs.nethereum.com/en/latest/introduction/web3/

![image](https://user-images.githubusercontent.com/6970347/135576394-c6cfa409-0016-4fce-9d8e-9be91641d5ce.png)

Am I missing anything?

**Comment by juanfranblanco** [comment]
Yes it is no in the latest version as we have 1559, Legacy and possible othe transactions types. The documentation needs to be updated.

You can do this (for Legacy transactions as it includes the gas price)
https://github.com/Nethereum/Nethereum/blob/master/tests/Nethereum.Accounts.IntegrationTests/SignOfflineTransactionTest.cs#L25-L44

or just, you mainly have to include all the fields, to avoid making any external calls. 

```web3.Eth.TransactionManager.SignTransactionAsync```

**Comment by juanfranblanco** [comment]
Another option is to use the "AccountOfflineTransactionSigner" direclty calling SignTransaction(Account account, TransactionInput transaction)
